money_test.go

```diff
@@ -73,39 +73,76 @@ func TestMulYen(t *testing.T) {
 }
 
 func TestDiv(t *testing.T) {
-	// Rational number with repeating decimals get truncated to 2 decimals
-	m1 := Money{1000, "EUR"}
-	m2 := Money{300, "EUR"}
-	m3 := m1.Div(m2)
-	if m3.Get() != 3.33 {
-		t.Errorf("expected money amount to be %v, got %v", 3.33, m3.Get())
+	tests := []struct {
+		name      string
+		money1    Money
+		money2    Money
+		expResult Money
+		expPanic  bool
+	}{
+		{
+			name:      "Rational number with repeating decimals get truncated to 2 decimals",
+			money1:    Money{1000, "EUR"},
+			money2:    Money{300, "EUR"},
+			expResult: Money{333, "EUR"},
+		},
+		{
+			name:      "Rational number with repeating decimals get truncated and rounded to 2 decimals",
+			money1:    Money{2000, "EUR"},
+			money2:    Money{300, "EUR"},
+			expResult: Money{667, "EUR"},
+		},
+		{
+			name:      "Rational number with repeating decimals get truncated and rounded to 0 decimals",
+			money1:    Money{2000, "JPY"},
+			money2:    Money{300, "JPY"},
+			expResult: Money{7, "JPY"},
+		},
+		// PAY-355: three tests below are testing the workaround for finding per pay period amounts!
```

Comment:
I don't think this comment makes sense in the context of this repo!

Reply:
You are right - copypasta, my mistake! Will update.